eZ/Bundle/PlatformInstallerBundle/src/Installer/DbBasedInstaller.php

```diff
@@ -67,4 +73,36 @@ protected function runQueriesFromFile($file)
             $this->db->exec($query);
         }
     }
+
+    /**
+     * Get DBMS-specific SQL data file path.
+     *
+     * @param string $relativeFilePath SQL file path relative to {baseDir}/{dbms} directory
+     *
+     * @return string absolute existing file path
+     *
+     * @throws \eZ\Publish\Core\Base\Exceptions\InvalidArgumentException
+     *
+     * @since 6.13
+     */
+    final protected function getDatabasePlatformSpecificSQLFile($relativeFilePath)
```

Comment:
Just curious, is there a need for `final` here?

Reply:
> Just curious, is there a need for `final` here?

Added due to [this request](https://github.com/ezsystems/ezpublish-kernel/pull/2153#discussion_r151973423) by @andrerom so he can probably answer this better. 
My implicit understanding of that change request is disallowing developer to interfere with internal paths for our installer data, but maybe there's more to this?

Reply:
Hmm. Would make sense I guess, but user still can call `getDatabasePlatformSpecificSQLFile('../../../../something.sql)` and ignore our internal path, while he could just override method. I don't mind `final` here. Just curiosity.

Reply:
Maybe also rename it to make it clear it is only for kernel files then